make_wheels.py

```diff
@@ -128,18 +160,37 @@ def write_ziglang_wheel(out_dir, *, version, platform, archive):
     import subprocess; sys.exit(subprocess.call(argv))
 '''.encode('ascii')
 
+    # Check if any license files are missing and warn. This
+    # is useful to detect changes in the Zig archive structure
+    # around licenses that may require updating the metadata.
+    missing_licenses = set(license_paths) - found_license_files
+    if missing_licenses:
+        print(f"\033[93mWarning: the following license files were not found in the Zig archive: {', '.join(sorted(missing_licenses))} "
+              "\nThis may indicate a change in Zig's license file structure or an error in the listing of license files and/or paths.\033[0m"
+            )
```

Comment:
What about going through all files in the tarball matching the regex `/license/i` and warning if any of them are missing from `license_paths`?

Reply:
I thought about it, it's not going to be robust enough to cover cases where the license is called "COPYRIGHT" or "COPYING. I left it alone after that, but a partial check sounds good to me.

Reply:
What about `/\b(license|copyright|copying)\b/i`?

Reply:
Sounds good!

Reply:
Addressed in 67b92005ccb83c347b8c4bc231058a3e7accb6ca. I added a larger pattern and the regex caught two more files – added them to the list! :)

Reply:
That's how you know it was working :D